colin.spec

```diff
@@ -9,13 +9,14 @@
 %endif
 
 Name:           %{pypi_name}
-Version:        0.3.0
-Release:        1%{?dist}
-Summary:        Tool to check generic rules/best-practices for containers/images/dockerfiles
+Version:        0.3.1.39.g2c5e46f
+Release:        4%{?dist}
```

Comment:
Why? It could be `1`. Version is changing `g68d7436` -> `g2c5e46f`.

Reply:
the spec file is copied from fedora rawhide dist-git, that's why release is 4

whoops; version is wrong, will change